sharding-spring/sharding-jdbc-orchestration-spring/sharding-jdbc-orchestration-spring-namespace/src/test/resources/META-INF/rdb/namespace/shardingMasterSlaveNamespace.xml

```diff
@@ -32,12 +32,12 @@
                         ">
 
     <import resource="../datasource/masterSlaveDataSource.xml" />
-    
-    <bean id="standardModuloDatabaseShardingAlgorithm" class="org.apache.shardingsphere.shardingjdbc.orchestration.spring.algorithm.StandardModuloDatabaseShardingAlgorithm" />
-    <bean id="standardModuloTableShardingAlgorithm" class="org.apache.shardingsphere.shardingjdbc.orchestration.spring.algorithm.StandardModuloTableShardingAlgorithm" />
-    <bean id="defaultComplexKeysShardingAlgorithm" class="org.apache.shardingsphere.shardingjdbc.orchestration.spring.algorithm.DefaultComplexKeysShardingAlgorithm" />
-    <bean id="defaultHintShardingAlgorithm" class="org.apache.shardingsphere.shardingjdbc.orchestration.spring.algorithm.DefaultHintShardingAlgorithm" />
-   
+
+    <spi:sharding-algorithm id="standardModuloDatabaseShardingAlgorithm" type="STANDARD_TEST_DB"/>
+    <spi:sharding-algorithm id="standardModuloTableShardingAlgorithm" type="STANDARD_TEST_TBL"/>
+    <spi:sharding-algorithm id="defaultComplexKeysShardingAlgorithm" type="complex"/>
```

Comment:
`complex` is not a proper type name, so could rename it in `defaultComplexKeysShardingAlgorithm ` as `COMPLEX_TEST`.

Reply:
> `complex` is not a proper type name, so could rename it in `defaultComplexKeysShardingAlgorithm ` as `COMPLEX_TEST`.

@tristaZero Okay, I will modify it.